string order_id
string pkg_location